mentii/mentii
# Edit class details

Allows teacher or admin to edit class title, department, section, or description

**Significant Changes**
1. Added a button to edit class details for teacher
2. Edit class details form
3. Changed `section` attribute in classes to `classSection` to prevent AWS conflict

**How to Test**
1. Login as admin or teacher
2. Create a class
3. Go to class details page and see **Edit Class** button in top right hand corner
4. When filling out a form and **Cancel** is hit, the form should repopulate with old data
5. Edit class details and hit **Save Changes**. Details page should update and in dashboards.

**Notes**
:zap:
